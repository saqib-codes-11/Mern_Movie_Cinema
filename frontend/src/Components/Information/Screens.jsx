import '../../CSS/Pages.css';
import { Card, Accordion, Button, Container, Jumbotron } from "react-bootstrap"

const Screens = () => {
    return (
        <div>
            <div class='background'>
                <Container>
                    <Jumbotron className="bgBlur">
                        <h1 class='landing-text'> Screens </h1>
                        <Card style={{ flex: 1, backgroundColor: '#A02626' }}>
                            <div class='row'>
                                <div class='col-sm'>
                                    <img src="./Images/StandardDecor.png" alt='Our Standard Cinema Screen Decor' width='100%' />
                                </div>
                                <div class='col-sm'>
                                    <h4> Standard Seating </h4>
                                    <p> Our Standard movie screens offer comfortable and spacious seating with enough room to accomodate over 100 guests. You can view our floorplan seating below to pick out your best </p>
                                </div>
                            </div>
                            <div class='container'>
                                <Accordion>
                                    <Card style={{ flex: 1, backgroundColor: '#912323' }}>
                                        <Card.Header>
                                            <Accordion.Toggle as={Button} variant="text" eventKey="0" variant="outline-dark" className="text-white">
                                                View Seating Plan
                                            </Accordion.Toggle>
                                        </Card.Header>
                                        <Accordion.Collapse eventKey="0">
                                            <Card.Body>
                                                <img src="./Images/StandardPlan.png" alt='Standard Seating Plan' width='100%' />
                                            </Card.Body>
                                        </Accordion.Collapse>
                                    </Card>
                                </Accordion>
                            </div>
                            <p />
                        </Card>
                        <p />
                        <Card style={{ flex: 1, backgroundColor: '#A02626' }}>
                            <div class='row'>
                                <div class='col-sm'>
                                    <h4> Deluxe Seating </h4>
                                    <p> Our world-renown IMAX Screens are perfect for any film lover who wants to sit back enjoy in comfort. With leather reclinable chairs, our mission is to give you the comfort you deserve allowing you to focus solely on the movie. Find our seating plan attached below. </p>
                                </div>
                                <div class='col-sm'>
                                    <img src="./Images/DeluxeDecor.png" alt='Our Deluxe Cinema Screen Decor' width='100%' />
                                </div>
                            </div>
                            <div class='container'>
                                <Accordion>
                                    <Card style={{ flex: 1, backgroundColor: '#912323' }}>
                                        <Card.Header>
                                            <Accordion.Toggle as={Button} variant="text" eventKey="0" variant="outline-dark" className="text-white">
                                                View Seating Plan
                                            </Accordion.Toggle>
                                        </Card.Header>
                                        <Accordion.Collapse eventKey="0">
                                            <Card.Body>
                                                <img src="./Images/DeluxePlan.png" alt='Deluxe Seating Plan' width='100%' />
                                            </Card.Body>
                                        </Accordion.Collapse>
                                    </Card>
                                </Accordion>
                            </div>
                            <p />
                        </Card>
                    </Jumbotron>
                </Container>

            </div>
        </div>
    )
}

export default Screens;
