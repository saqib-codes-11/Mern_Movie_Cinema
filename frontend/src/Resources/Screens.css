.row {
    padding: 8px;
}

.container input {
  width: 95%;
  clear: both;
  margin-bottom: 4px;
}

.container .ticket-type{
    margin: 5px;
    width: 50px;
}

.background .container {
    padding: 10px;
}

input:invalid+span:after {
    content: '✖';
    padding-left: 3px;
}

input:valid+span:after {
    content: '✓';
    padding-left: 3px;
}

form #submit {
    margin: 3px;
    margin-bottom: 8px;
}

input#email {
    width:80%;
}

textarea#message {
    width: 80%;
    height: 250px;
}