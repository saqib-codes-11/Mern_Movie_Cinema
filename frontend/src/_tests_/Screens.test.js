import React from 'react';
import renderer from 'react-test-renderer';
import Screens from '../Components/Information/Screens';

test("It matches the snapshot", () => {
    const tree = renderer.create(<Screens />).toJSON();
    expect(tree).toMatchSnapshot();

})