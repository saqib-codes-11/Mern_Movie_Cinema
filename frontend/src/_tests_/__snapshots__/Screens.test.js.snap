// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches the snapshot 1`] = `
<div>
  <div
    class="background"
  >
    <div
      className="container"
    >
      <div
        className="bgBlur jumbotron"
      >
        <h1
          class="landing-text"
        >
           Screens 
        </h1>
        <div
          className="card"
          style={
            Object {
              "backgroundColor": "#A02626",
              "flex": 1,
            }
          }
        >
          <div
            class="row"
          >
            <div
              class="col-sm"
            >
              <img
                alt="Our Standard Cinema Screen Decor"
                src="./Images/StandardDecor.png"
                width="100%"
              />
            </div>
            <div
              class="col-sm"
            >
              <h4>
                 Standard Seating 
              </h4>
              <p>
                 Our Standard movie screens offer comfortable and spacious seating with enough room to accomodate over 100 guests. You can view our floorplan seating below to pick out your best 
              </p>
            </div>
          </div>
          <div
            class="container"
          >
            <div
              className="accordion"
            >
              <div
                className="card"
                style={
                  Object {
                    "backgroundColor": "#912323",
                    "flex": 1,
                  }
                }
              >
                <div
                  className="card-header"
                >
                  <button
                    className="text-white btn btn-outline-dark"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    View Seating Plan
                  </button>
                </div>
                <div
                  aria-expanded={null}
                  className="collapse"
                >
                  <div
                    className="card-body"
                  >
                    <img
                      alt="Standard Seating Plan"
                      src="./Images/StandardPlan.png"
                      width="100%"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
          <p />
        </div>
        <p />
        <div
          className="card"
          style={
            Object {
              "backgroundColor": "#A02626",
              "flex": 1,
            }
          }
        >
          <div
            class="row"
          >
            <div
              class="col-sm"
            >
              <h4>
                 Deluxe Seating 
              </h4>
              <p>
                 Our world-renown IMAX Screens are perfect for any film lover who wants to sit back enjoy in comfort. With leather reclinable chairs, our mission is to give you the comfort you deserve allowing you to focus solely on the movie. Find our seating plan attached below. 
              </p>
            </div>
            <div
              class="col-sm"
            >
              <img
                alt="Our Deluxe Cinema Screen Decor"
                src="./Images/DeluxeDecor.png"
                width="100%"
              />
            </div>
          </div>
          <div
            class="container"
          >
            <div
              className="accordion"
            >
              <div
                className="card"
                style={
                  Object {
                    "backgroundColor": "#912323",
                    "flex": 1,
                  }
                }
              >
                <div
                  className="card-header"
                >
                  <button
                    className="text-white btn btn-outline-dark"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    View Seating Plan
                  </button>
                </div>
                <div
                  aria-expanded={null}
                  className="collapse"
                >
                  <div
                    className="card-body"
                  >
                    <img
                      alt="Deluxe Seating Plan"
                      src="./Images/DeluxePlan.png"
                      width="100%"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
          <p />
        </div>
      </div>
    </div>
  </div>
</div>
`;
